docs/docs/recipes/working-with-images.md

```diff
@@ -1,73 +1,75 @@
 ---
-title: "Recipes: Working with Images"
+Titre: "Recettes: travailler avec des images"
 tableOfContentsDepth: 1
 ---
 
-Access images as static resources, or automate the process of optimizing them through powerful plugins.
+Accédez aux images en tant que ressources statiques ou automatisez le processus de leur optimisation grâce à de puissants plugins.
 
-## Import an image into a component with webpack
+## Importer une image dans un composant avec webpack
 
-Images can be imported right into a JavaScript module with webpack. This process automatically minifies and copies the image to your site's `public` folder, providing a dynamic image URL for you to pass to an HTML `<img>` element like a regular file path.
+Les images peuvent être importées directement dans un module JavaScript avec webpack. Ce processus réduit et copie automatiquement l'image dans le dossier `public` de votre site, en fournissant une URL d'image dynamique que vous pouvez passer à un élément HTML `<img>` comme un chemin de fichier normal.
 
 <EggheadEmbed
   lessonLink="https://egghead.io/lessons/gatsby-import-a-local-image-into-a-gatsby-component-with-webpack"
   lessonTitle="Import a Local Image into a Gatsby Component with webpack"
 />
 
-### Prerequisites
+### Conditions préalables
 
-- A [Gatsby Site](/docs/quick-start) with a `.js` file exporting a React component
-- an image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) in the `src` folder
+- Un [Site Gatsby](/docs/quick-start) avec un fichier `.js` exportant un composant React
+- une image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) dans le dossier `src`
 
-### Directions
+### Les directions
 
-1. Import your file from its path in the `src` folder:
+1. Importez votre fichier depuis son chemin dans le dossier `src`:
 
 ```jsx:title=src/pages/index.js
 import React from "react"
 // Tell webpack this JS file uses this image
 import FiestaImg from "../assets/fiesta.jpg" // highlight-line
 ```
 
-2. In `index.js`, add an `<img>` tag with the `src` as the name of the import you used from webpack (in this case `FiestaImg`), and add an `alt` attribute [describing the image](https://webaim.org/techniques/alttext/):
+2. Dans `index.js`, ajoutez une balise `<img>` avec le `src` comme nom de l'importation que vous avez utilisé à partir de webpack (dans ce cas `FiestaImg`), et ajoutez un attribut `alt` [décrivant l'image] (https://webaim.org/techniques/alttext/):
 
 ```jsx:title=src/pages/index.js
 import React from "react"
 import FiestaImg from "../assets/fiesta.jpg"
 
-export default () => (
-  // The import result is the URL of your image
-  <img src={FiestaImg} alt="A dog smiling in a party hat" /> // highlight-line
-)
+export default function Home() {
```

Comment:
Please don't change functions

Reply:
I dont see any change here i have kept as it is

export default function Home() {
  return (
    // The import result is the URL of your image
    <img src={FiestaImg} alt="A dog smiling in a party hat" /> // highlight-line
  )
}

Reply:
La ligne 38 a bien été changée après la traduction, merci de remettre l'ancienne
Idem avec le return qui a été ajouté.